instrumentation/okhttp/okhttp-3.0/library/src/main/java/io/opentelemetry/instrumentation/okhttp/v3_0/OkHttpTelemetryBuilder.java

```diff
@@ -56,7 +47,7 @@ public OkHttpTelemetryBuilder addAttributesExtractor(
    */
   @CanIgnoreReturnValue
   public OkHttpTelemetryBuilder setCapturedRequestHeaders(List<String> requestHeaders) {
-    httpAttributesExtractorBuilder.setCapturedRequestHeaders(requestHeaders);
+    this.capturedRequestHeaders = requestHeaders;
```

Comment:
unnecessary `this`?

Reply:
nit: reassigning the reference allows the caller to later manipulate the list. It's silly/pedantic for a builder, I know, but prefer making the field final and just adding the to list. Could even rename to `addCapturedRequestHeaders()` if that's nicer (and avoids a clear/add race).

Same applies to response headers.

Reply:
> reassigning the reference allows the caller to later manipulate the list.

Great point! Will fix that, thanks

> but prefer making the field final and just adding the to list. Could even rename to `addCapturedRequestHeaders()` if that's nicer (and avoids a clear/add race).

I think at some point we've decided to have a setter method (not an "adder") in all HTTP instrumentations, as it resulted in simpler API. We can revisit that discussion, but we'd have to do it for all HTTP instrumentations.